pkg/manager/manager_table.go

```diff
@@ -25,7 +28,17 @@ func (sm *Manager) startTableMode() error {
 	// want to step down
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	log.Infof("all routing table entries will exist in table [%d]", sm.config.RoutingTableID)
+	log.Infof("all routing table entries will exist in table [%d] with protocol [%d]", sm.config.RoutingTableID, sm.config.RoutingProtocol)
+
+	go func() {
+		// we assume that after 10s all services should be configured so we can delete redundant routes
+		time.Sleep(time.Second * 10)
```

Comment:
Just a thought, since here it make an assumption, is it safer to turn if off by default and allow user to turn it on if needed?

Reply:
OK, I've added flag `cleanRoutingTable` which, if true will enable this cleaning and defaults to false.

TBH, I don't like this assumption at all, but I think there's no way to tell if all the services were already processed. At least I cannot see it. :(